src/components/shared/ProductDefaultImage.tsx

```diff
@@ -0,0 +1,48 @@
+import {Image, ImageProps} from "@chakra-ui/react"
+import {useMemo, FC} from "react"
+import {ILineItemProduct} from "types/ordercloud/ILineItemProduct"
+import {IProduct} from "types/ordercloud/IProduct"
+
+const PRODUCT_DEFAULT_IMAGE__FALLBACK_URL = "/raster/dummy-image-square.jpg"
+
+interface IProductDefaultImage extends Omit<ImageProps, "alt" | "src" | "title"> {
+  product: IProduct | ILineItemProduct
+  preferThumbnail?: boolean
```

Comment:
I think `thumbnail` would have been good enough here but this works!

Reply:
I imagine later on we may need to support various sizes if this component is used in more than one place. I would also assume that using the preferThumbnail = true setting would have adjusted the image size. But again, if this is working for now, we can add more settings & adjust these options later.

Reply:
NVM, I see now that you are extending ImageProps and passing the "rest" of them to the Image component.